src/main/java/edu/hm/hafner/analysis/IssueBuilder.java

```diff
@@ -511,6 +517,12 @@ public Issue build() {
      * @return the created issue
      */
     public Issue buildAndClean() {
+        if (lineRanges != null && lineRanges.size()>1) {
```

Comment:
Please fix warnings before requesting a review

Reply:
I mentioned this in the comment above as to why I used this. If this doesn't work, then

```
assert lineRanges != null;
        if (!lineRanges.isEmpty()) {...
```
is this okay, to prevent `NullPointerException`?